package.json

```diff
@@ -8,8 +8,10 @@
     "node": "4.1.x"
   },
   "scripts": {
-    "start": "node index.js",
-    "test": "mocha test/*_test.js test/**/*_test.js"
+    "start": "node -r babel-core/register -r dotenv/config lib/index.js",
+    "prepublish": "babel lib -d dist",
```

Comment:
:+1: That makes sense.

Alternatively we could close the gap between dev and prod even further and use babel-require in the entry file and not bother with compilation.

http://stackoverflow.com/questions/31622802/can-i-use-babel-require-hook-in-production-for-es6